app/models/post.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from app.models.abstracts import Date
+from app.models import User
+from app.managers import UserManager
+
+class Post(Date):
+    class Meta:
+        db_table = 'posts'
+
+    REQUIRED_FIELDS = ['content']
```

Comment:
I think this can be removed since not specifying required=True or required=False when initialising a field renderds the field required. For example

user = Models.ForeignKey(User, .... ) makes the user fk required but

user = Models.foreignKey(User ,... , required=false) creates a nullable fk in the database.